spec/routes/donations_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spec_helper.rb'
+require_relative 'app_helper.rb'
 
```

Comment:
why `app_helper.rb` and not just `app_helper`?

Reply:
good catch